src/io/flutter/run/daemon/DaemonConsoleView.java

```diff
@@ -81,12 +81,6 @@ public void print(@NotNull String text, @NotNull ConsoleViewContentType contentT
       return;
     }
 
-    // "flutter test" prints this when using --start-paused.
-    // (We should probably change it to stop printing it in machine mode.)
-    if (trimmed.startsWith("You should first set appropriate breakpoints, then resume the test in the debugger.")) {
-      return;
-    }
-
```

Comment:
cc @devoncarew whose thought a bit about querying properties/capabilities IIRC.

In the short term maybe something coarse like SDK number will suffice.